python/lsst/summit/utils/utils.py

```diff
@@ -1419,3 +1425,221 @@ def getSunAngle(time: Time | None = None) -> float:
         time = Time.now()
     sun_altaz = get_sun(time).transform_to(AltAz(obstime=time, location=SIMONYI_LOCATION))
     return sun_altaz.alt.deg
+
+
+@dataclass
+class RobustFitResult:
+    intercept: float
+    slope: float
+    scatter: float
+    slope_pvalue: float
+    slope_stderr: float
+    slope_tvalue: float
+    intercept_pvalue: float
+    intercept_stderr: float
+    intercept_tvalue: float
+
+
+class RobustFitter:
+    """
+    Robust linear fit using RANSAC,
+    with confidence and prediction intervals from statsmodels.
```

Comment:
Docstring should start with a full line i.e. no premature line break, and have the parameters and returns etc. Things like `min_samples` really need explaining, as I wouldn't have thought, from the variable name, that it would be a float, for example.

Reply:
Got it